.github/workflows/release_validation.yml

```diff
@@ -0,0 +1,21 @@
+name: release_validation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  cocoapods_lib_lint:
+    name: cocoapods_lib_lint
+    runs-on: macos-11
+    timeout-minutes: 120
```

Comment:
Does the timeout has to be so high? Just curious

Reply:
No, just copy pasted from a different action. Reduced to 20. The last run took 5 minutes, so 4x headroom should be plenty.